features/isnotwhitespace.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Define a dimension as a range of values from a cell reference but not including the cells containing whitespace.
+  I want to be able to define a dimension as a range of values from a cell reference, but I want to
+  ignore any whitespace cells that lie within that range.
+
+  Scenario: Define non-whitespace year from a range of cell references ignoring any whitespace cells.
+    Given we load an xls file named "bulletindataset2v2.xls"
+    And select the sheet "Table 2a"
+    And we define cell selections as
+      | key             | value                                             |  
+      | unit            | tab.excel_ref("A11:A250").is_not_whitespace()     |
+    Then we confirm year contains no blank cells.
```

Comment:
I really wanna just approve hits but I think this is an old approach leftover and reads a bit odd, I'm guessing i needs to be `we confirm "unit" contains no blank cells`.

in the step you can do:

```
@then(u'we confirm "{selection_name}" contains no blank cells.')
def step_impl(context, selection_name):
    bag_of_cells = context.selections[selection_name]
    # check whatever it is you need to check about the "bag_of_cells" selection
```

Reply:
Great spot, knew I forgot something. 
I have changed the step to refer to `selection name` which is in keeping with the existing steps. The actual name can always be found from the key in the table attached to the definition step. I have changed most of these to reflect this.